netty/src/main/java/io/grpc/netty/HandlerSettings.java

```diff
@@ -23,9 +23,12 @@
 /**
  * Allows autoFlowControl to be turned on and off from interop testing and flow control windows to
  * be accessed. For internal use only.
+ *
+ * @deprecated renamed to {@link InternalHandlerSettings} and should not be used externally
  */
 @VisibleForTesting // Visible for tests in other packages.
 @Internal
+@Deprecated
 public final class HandlerSettings {
```

Comment:
Then let's make it clear this implementation is useless and calling it does nothing. Let's remove the package-private and private stuff and replace the public pieces with fakes. And then change the docs to say it does nothing.